app/pb_kits/playbook/pb_highlight/_highlight.jsx

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+/* eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+import Highlighter from 'react-highlight-words'
+import React from 'react'
+
+type HighlightProps = {
+  className?: String,
+  data?: String,
+  id?: String,
+  children?: Array<React.ReactChild>,
```

Comment:
Did you consider using [React.Node](https://flow.org/en/docs/react/children) for the type of `children`?